### ISSUE TITLE
How to import and instantiate a new circos in ES6?

### ISSUE THREAD

**Issue by imranariffin** [nicgirault/circosJS#38]
Hi @nicgirault, 
I couldn't any information on README on how to use circosjs in ES6, so I tried this:

```
import React, { Component } from 'react';
import Circos from 'circos';

class Chart extends Component {
  render() {
    let myCircos = new Circos({
      container: '#lineChart',
      width: 500,
      height: 500,
    });
    return (
        <div id="lineChart"></div>
    );
  }
}
```

But I got this error:
```
TypeError: __WEBPACK_IMPORTED_MODULE_2_circos___default.a is not a constructor
```

I am very new to ES6 so please bear with me. Thank you very much, would appreciate any help.

**Comment by ferrantejake** [comment]
A quick look at [the source code](https://github.com/nicgirault/circosJS/blob/master/src/circos.js#L107) shows Circos is not a class, thus cannot be instantiated with the `new` keyword. It is a function and must be called as one. The code you are seeking looks like this:
```
    let myCircos = Circos({
      container: '#lineChart',
      width: 500,
      height: 500,
    });
```

**Comment by imranariffin** [comment]
@ferrantejake thank you on your comment. However, I've tried doing that and it gave me this error instead:

```
TypeError: __WEBPACK_IMPORTED_MODULE_2_circos___default(...) is not a function
```

Any idea?

**Comment by ferrantejake** [comment]
My apologies, I missed your import statement. You are attempting to do a default import (see [this MDN article on importing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [this MDN article on exporting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default)). Given the way Circos is exported,  you want to import the library using the following
```
import * as Circos from 'circos'
```
This will import the function as the variable `Circos` which you can call using the specified code above.

**Comment by imranariffin** [comment]
@ferrantejake It's still giving me the same error. I'm not sure but could it be that Circos is exported as
```
// in ~/src/circos.js
const Circos = (conf) => {
  const instance = new Core(conf)
  return instance
}
module.exports = Circos
```
so
```
import 
``` 
doesn't work?

**Comment by ferrantejake** [comment]
Can you post your updated code? 

I wrote an example which demonstrates the intended usage. Here is a screenshot of the example being run. 
![image](https://user-images.githubusercontent.com/9873344/28487885-a00ca876-6e69-11e7-982c-044311fafb2b.png)

**Comment by imranariffin** [comment]
Here they are. My apologies for the delay.

![circos-debug-github-discussion](https://user-images.githubusercontent.com/5580715/28488473-db7721ac-6e78-11e7-8efb-66a14d63d857.png)

![circos-debug-github-discussion-error-page](https://user-images.githubusercontent.com/5580715/28488482-06a071ee-6e79-11e7-8e67-5bd2b5452ccf.png)

**Comment by nicgirault** [comment]
@imranariffin thanks for reporting this issue. I think the problem comes from the webpack configuration that expose that specify:

```
library: 'Circos',
libraryTarget: 'var'
```

I'll try to fix this ASAP

**Comment by nicgirault** [comment]
This issue should be fixed by upgrading to circos 2.0.7 Can you reopen this issue if it's not the case for you? Thanks

**Comment by imranariffin** [comment]
@nicgirault Thank you very much for the response. I'll open a new issue.
